types/node/events.d.ts

```diff
@@ -75,9 +75,37 @@ declare module "events" {
          */
         captureRejections?: boolean | undefined;
     }
-    interface StaticEventEmitterOptions {
```

Comment:
Are you sure this name is safe to change? This is an exported interface, after all.

Reply:
I concur, I don't think it can be removed, people are, for example, doing this (search on the GitHub will give lots of results):
![image](https://github.com/DefinitelyTyped/DefinitelyTyped/assets/14539/c85157eb-b7ca-4c04-897d-761db0646e05)

Reply:
Thanks, that's really a good point. I'll fix it so we don't make any breaking changes to exposed interfaces.